### ISSUE TITLE
Use better error message on REGISTER/SAREGISTER failure

### ISSUE THREAD

**Issue by DanielOaks** [oragono/oragono#1054]
When an account gets unregistered, we reserve it indefinitely ('til someone uses `/NS ERASE` to purge the account). If it's in that reserved state and someone tries to register it (or an oper tries to saregister it) we should tell them that it's in the reserved state and they need to `ERASE` the account.

**Comment by slingamn** [comment]
Makes sense, but I don't want to encourage the use of ERASE --- it's just an escape hatch.
